packages/mds-compliance-service/repository/entities/compliance-snapshot-entity.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2021 City of Los Angeles
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Entity, Column } from 'typeorm'
+import { BigintTransformer, IdentityColumn, RecordedColumn } from '@mds-core/mds-repository'
+import { ComplianceSnapshotDomainModel } from '../../@types'
+
+export interface ComplianceSnapshotEntityModel extends IdentityColumn, RecordedColumn {
+  compliance_snapshot_id: ComplianceSnapshotDomainModel['compliance_snapshot_id']
+  compliance_as_of: ComplianceSnapshotDomainModel['compliance_as_of']
+  provider_id: ComplianceSnapshotDomainModel['provider_id']
+  policy_name: ComplianceSnapshotDomainModel['policy']['name']
+  policy_id: ComplianceSnapshotDomainModel['policy']['policy_id']
+  excess_vehicles_count: ComplianceSnapshotDomainModel['excess_vehicles_count']
+  total_violations: ComplianceSnapshotDomainModel['total_violations']
+  vehicles_found: ComplianceSnapshotDomainModel['vehicles_found']
+}
```

Comment:
Didn't we decide to decouple the entity model definitions from the domain models?

Reply:
Yeah, the compliance rework pre-dated that decision though. Worth marking as tech debt.